src/SharpGLTF.Toolkit/Geometry/VertexTypes/VertexEmpty.cs

```diff
@@ -58,6 +58,10 @@ namespace SharpGLTF.Geometry.VertexTypes
 
         void IVertexMaterial.SetTexCoord(int index, Vector2 coord) { throw new ArgumentOutOfRangeException(nameof(index)); }
 
+        VertexMaterialDelta IVertexMaterial.Subtract(IVertexMaterial baseValue) { throw new NotSupportedException(); }
+
+        void IVertexMaterial.Add(in VertexMaterialDelta delta) { throw new NotSupportedException(); }

```

Comment:
Should do nothing:  { /* do nothing */ }

Reply:
done